bundles/AdminBundle/Controller/Rest/Element/DataObjectController.php

```diff
@@ -20,6 +20,7 @@
 use Pimcore\Model\DataObject;
 use Pimcore\Model\Webservice\Data\DataObject\Concrete\In as WebserviceObjectIn;
 use Pimcore\Model\Webservice\Data\DataObject\Concrete\Out as WebserviceObjectOut;
+use Pimcore\Model\Webservice\Data\DataObject\Concrete\Out;
```

Comment:
This is `use`d twice. See also line 133.

Reply:
thx! 👍